NewSpring/crete
# feat: Add option to remove prayer answer

## DESCRIPTION

### What does this PR do, or why is it needed?
![Screenshot 2020-05-05 10 22 49](https://user-images.githubusercontent.com/3229463/81077031-710d9c00-8eba-11ea-915f-2f0ed532b971.png)

This PR adds a `removeAnswer` mutation so that a person can remove the answer to their prayer if they need to.

### How do I test this PR?
Run the `removeAnswer` mutation in your GraphQL playground of choice. It should set the `answer` field to `null`.
```
mutation {
  removeAnswer(id: "Prayer:d82359939f90bb2c6a528b69294231c9") {
    id
    text
    answer
  }
}
```

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
